### PR TITLE
Fix success response code on jpeg images

### PR DESCRIPTION
**Pull request by YarikMamykin**
Implements/Fixes 
[#319](https://github.com/smartdevicelink/sdl_hmi/issues/319)
[#321](https://github.com/smartdevicelink/sdl_hmi/issues/321)
[#329](https://github.com/smartdevicelink/sdl_hmi/issues/329)

This PR is **ready** for review.

**WARNING! This PR should be merged only after https://github.com/smartdevicelink/sdl_hmi/pull/344**
Changes in this PR are based on changes from PR [344](https://github.com/smartdevicelink/sdl_hmi/pull/344)

### Testing Plan
Manual testing

### Summary
Enhanced image validation function and added check for template non-png images.

### CLA
- [X] I have signed [the CLA](https://docs.google.com/forms/d/e/1FAIpQLSdsgJY33VByaX482zHzi-xUm49JNnmuJOyAM6uegPQ2LXYVfA/viewform)

### CODE REVIEW

**Comment by YarikMamykin** [comment]
@theresalech please notice that this PR is ready for Livio Team Code Review.

File: app/view/sdl/AlertPopUp.js [hunk 1]
```diff
@@ -274,6 +274,17 @@ SDL.AlertPopUp = Em.ContainerView.create(
                                                                         // popUp
       this.set('priority', priority);
       clearTimeout(this.timer);
+
+      var callback = function(failed) {
+        self.reason = 'WARNINGS';
```
Comment (jacobkeeler) [inline app/view/sdl/AlertPopUp.js:279]:
```suggestion
        self.reason = failed ? 'WARNINGS' : 'SUCCESS';
```

File: ffw/UIRPC.js [hunk 2]
```diff
@@ -274,9 +292,31 @@ FFW.UI = FFW.RPCObserver.create(
...
+
+            var imageList = [];
+            if(request.params.menuIcon) {
+              imageList.push(request.params.menuIcon.value);      
+            }
+
+            if(request.params.vrHelp) {
+              for(var i = 0; i < request.params.vrHelp.length; i++) {
+                if(request.params.vrHelp[i].image) {
+                  imageList.push(request.params.vrHelp[i].image.value);
```
Comment (jacobkeeler) [inline ffw/UIRPC.js:304]: Should there be an `isPng` check for the images in this function?

**Review by jacobkeeler — CHANGES_REQUESTED** [review]

**Comment by jacobkeeler** [comment]
@YarikMamykin also fix merge conflicts

**Review by iCollin — COMMENTED** [review]
should this be case - insensitive?

edit: i’m not reviewing this PR just had this comment saved from taking a look previously

**Comment by AKalinich-Luxoft** [comment]
@iCollin @jacobkeeler because of a lot of conflicts and some architectural changes introduced in release 7.0 it makes sense to completely change this fix.
Closed in favor of #462

**Pull request closed without merging**